### PR TITLE
init readme

### PR DESCRIPTION
**Pull request by kittyandrew**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1 +1,30 @@
-# waiting for andrew to pr a good readme
\ No newline at end of file
+About
```
Comment (tulir) [inline README.rst]: Usually the readme should start with the project name

```suggestion
telethon-tgcrypto
```

File: README.rst [hunk 2]
```diff
@@ -1 +1,29 @@
-# waiting for andrew to pr a good readme
\ No newline at end of file
+telethon-tgcrypto
+-----
+
+Module ``telethon-tgcrypto`` brings better performance to
+`Telethon <https://github.com/LonamiWebs/Telethon>`__ lib.
+
+Installation
+------------
```
Comment (tulir) [inline README.rst]: This and credits should probably be smaller headers, but I'm not sure how RST works exactly

**Comment by kittyandrew** [comment]
https://github.com/kittyandrew/telethon-tgcrypto this better?

File: README.rst [hunk 3]
```diff
@@ -1 +1,29 @@
...
+telethon-tgcrypto
+*****************
+
+Module ``telethon-tgcrypto`` brings better performance to
+`Telethon <https://github.com/LonamiWebs/Telethon>`__ lib.
+
+Installation
+------------
+
+Note: you must not have ``cryptg`` installed, because it interfiers with
```
Comment (tulir) [inline README.rst]:
```suggestion
**N.B.** you must not have ``cryptg`` installed, because it interferes with
```

**Review by tulir — APPROVED** [review]

**Pull request merged**